### DISCUSSION TITLE
Extract cookies / localStorage from Chrome

### DISCUSSION THREAD

**Discussion by jamiehaywood** [johnlindquist/kit#725] [Ideas]
I'm not sure if it's possible, but extracting the cookies and localStorage would be immensely useful for being able to authenticate to various services 😄

**Comment by johnlindquist** [comment]
This should put you on the right path:

https://github.com/johnlindquist/kit/discussions/629#discussioncomment-1725659

Can you share your idea? Really curious what you have in mind

**Reply by jamiehaywood** [reply]
Sure! At work there are a lot of systems that require SAML SSO for access (timesheets, change requests etc.) Until now, I've just written headless chrome scripts to do stuff, but that's quite heavyweight. Being able to hijack the session cookie with ScriptKit and run the requests using `fetch` would be awesome, and would be much lighter weight :)
